src/TaskRunner/Commands/ComponentCheckCommands.php

```diff
@@ -182,10 +183,18 @@ function ($value) {
             $this->io()->success('Components checked, nothing to report.');
         } else {
             $this->io()->note([
-                'NOTE: It is possible to bypass the insecure and outdated check by providing a token in the commit message.',
-                'The available tokens are:',
-                '    - [SKIP-OUTDATED]',
-                '    - [SKIP-INSECURE]',
+                'NOTE: It is possible to bypass the insecure and outdated check:',
+                '- Insecure check:',
+                '   - by providing a token in the commit message: [SKIP-INSECURE]',
+                '- Outdated check:',
+                '   - by providing a token in the commit message: [SKIP-OUTDATED]',
+                '   - Or, updating runner.yml.dist toolkit settings: ',
+                '       toolkit:',
+                '         ...',
+                '         components:',
+                '           outdated:',
+                '             check: false',
+                '         ...',
```

Comment:
Hi @crgomes ,
What do you think about changing the message to something like this.
```suggestion
                'It is possible to bypass the insecure and outdated check:',
                '- Insecure check:',
                '   - by providing a token in the commit message: [SKIP-INSECURE]',
                '- Outdated check:',
                '   - by providing a token in the commit message: [SKIP-OUTDATED]',
                '   - Or, update the configuration in the runner.yml.dist as shown below: ',
                '        toolkit:',
                '          components:',
                '            outdated:',
                '              check: false',
```